src/main/webapp/.prettierignore

```diff
@@ -0,0 +1,3 @@
+# Ignore artifacts:
+build
+coverage
```

Comment:
I am not sure if this file should be included here

Reply:
I'm not sure if it does anything, so yeah, deleted it.